brave/brave-ios
# Fix #3568: Add Japanese default favorites.

## Summary of Changes


This pull request fixes #3568 

## Submitter Checklist:


## Test Plan:

- Set device region to Japan
- Fresh install 1.29
- Verify that favorites from the ticket are added

## Screenshots:


<img width="335" alt="Zrzut ekranu 2021-07-16 o 15 03 06" src="https://user-images.githubusercontent.com/6950387/125952268-3a6e44d3-2bc1-4782-9162-309aaa53905f.png">


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
